### PR TITLE
Touch up Ubuntu install instructions

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Touch up the install instructions for Ubuntu and wraps up
https://github.com/containers/podman/issues/7268

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@lsm5 PTAL

File: getting-started/installation.md [hunk 1]
```diff
@@ -194,13 +194,16 @@ sudo yum module install -y container-tools:1.0
...
+The Kubic project provides packages for Ubuntu 18.04, 19.04, 19.10 and 20.04.  **NOTE:** The `sudo apt update`
+in the below example has worked in some instances and it seems to be required on those machines, but it has
+not worked on other machines and it wasn't required to install Podman.
 
 ```bash
 . /etc/os-release
 echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
 curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/Release.key | sudo apt-key add -
 sudo apt-get update -qq
+sudo apt update
```
Comment (lsm5) [inline getting-started/installation.md]: this is weird, both `apt-get update` and `apt update` should produce the same result. Do you have any background info or links to github issues?

Comment (lsm5) [reply getting-started/installation.md]: ah I see the linked issue ... looking..

Comment (lsm5) [reply getting-started/installation.md]: I guess @aplatypus meant that `sudo apt-get upgrade` is required. The `update` option only refreshes repository and package metadata while the `uprade` option actually upgrades the packages.

While I didn't seem to need `upgrade` on my 20.04 container before installing podman, I'm not opposed to it.

So, just a `s/update/upgrade/` , and maybe prefer either of `apt` or `apt-get` for consistency and use it throughout the document, then I can LGTM. Also, I'm cool with `-qq` being removed as mentioned in the linked issue.

Comment (TomSweeneyRedHat) [reply getting-started/installation.md]: ugh, dyslexic moment on my end with update/upgrade.  Thanks for the look.  I personally prefer the -qq, but I seem to be in the minority, removed.  Reposting shortly.

File: getting-started/installation.md [hunk 2]
```diff
@@ -194,14 +194,17 @@ sudo yum module install -y container-tools:1.0
...
+not worked on other machines and it wasn't required to install Podman.
 
 ```bash
 . /etc/os-release
 echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
 curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/Release.key | sudo apt-key add -
-sudo apt-get update -qq
-sudo apt-get -qq -y install podman
+sudo apt-get update
+sudo apt-get upgrade
```
Comment (lsm5) [inline getting-started/installation.md]: sorry, let's make it `sudo apt-get -y upgrade`

Comment (lsm5) [reply getting-started/installation.md]: @TomSweeneyRedHat ^ .. after that, LGTM.

**Comment by TomSweeneyRedHat** [comment]
Added the '-y and repushed.

**Review by lsm5 — APPROVED** [review]
LGTM

**Pull request merged**